pkg/analytics/analytics_test.go

```diff
@@ -0,0 +1,39 @@
+package analytics_test
+
+import (
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/windmilleng/wmclient/pkg/analytics"
+)
+
+func TestFlush(t *testing.T) {
+	f := newAnalyticsFixture(t)
+	for i := 0; i < 10; i++ {
+		f.a.Incr("event", nil)
+	}
+
+	f.a.Flush(time.Second)
+	if len(f.reqs) != 10 {
+		t.Errorf("Expected 10 events sent. Actual: %d", len(f.reqs))
+	}
+}
+
+type analyticsFixture struct {
+	t    *testing.T
+	a    analytics.Analytics
+	reqs []*http.Request
+}
+
+func newAnalyticsFixture(t *testing.T) *analyticsFixture {
+	f := &analyticsFixture{t: t}
+	a := analytics.NewRemoteAnalytics(f, "test-app", "/report", "random-user", true)
+	f.a = a
+	return f
+}
+
+func (f *analyticsFixture) Do(req *http.Request) (*http.Response, error) {
```

Comment:
is this method used?

Reply:
ya! it's fulfilling the http.Client interface of analytics